Packs/Legacy/Integrations/Tanium/Tanium_test.py

```diff
@@ -0,0 +1,76 @@
+import demistomock as demisto
+
+CSV_STRING_1 = "header1,header2,header3\r\ncol1,col2,col3\r\ncol1,col2,col3"
+CSV_STRING_2 = "header1,header2,header3\r\ncol1,\"col2_1\r\ncol2_2\",col3\r\ncol1,col2,col3"
+CSV_STRING_3 = "header1,header2,header3"
```

Comment:
have these in multiline - or in a separate file altogether

Reply:
done